### PR TITLE
Introduction colors

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
Just made an intro and colors page .
Please let me know your feedback.

### CODE REVIEW

File: pages/colors/colors.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      <section class="section">
+        <article class="section-content underline">
+          <h2>starUI's Color Scheme</h2>
+          <p class="small-text">starUI has pre-defined color pallete which is compatible with all it's components. The colors are used in a generic basis which can be accessed using pre-defined classnames.</p>
+        </article>
+        <article class="section-content underline">
+          <h2>Default Palette</h2>
+          <p class="small-text">Below is the color palette of starUI along with the hex code.</p>
+          <div class="color-example">
+            <div class='primary'>
```
Comment (VishalPatil18) [inline pages/colors/colors.html:54]: **FIX**: You can bring these colors in the color palette in a single line. It looks a bit off having 2 colors below and 3 above

Screenshot 2022-01-27 at 21 23 44
.

File: index.html [hunk 2]
```diff
@@ -23,8 +23,8 @@ <h2>starUI🌟</h2>
   <main class='container-center'>
     <h2>Build fast, responsive sites with Bootstrap </h2>
     <p class="small-text">starUI is a free, open-source framework that provides ready-to-use frontend components that you can easily combine to build responsive interface. </p>
-    <button class='btn primary-btn small-text'><a>Get Started</a></button>
-    <button class='btn secondary-btn small-text'><a>GitHub</a></button>
+    <button class='btn primary-btn small-text'><a href="./pages/introduction/introduction.html">Get Started</a></button>
+    <button class='btn secondary-btn small-text'><a>GitHub</a></button>s
```
Comment (VishalPatil18) [inline index.html]: **Fix**: Remove the "s" after the githug button.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
There are a couple of fixes. Apart from that everything looks cool!

File: index.html (hunk 2 above)
Comment (Kevin-Solomon) [reply index.html]: Thank you Vishal !!

**Comment by AnkurChunekar** [comment]
Great work, Kevin. 
Your homepage, introduction and colors pages look awesome, code is well indented, comments are used properly. keep going.

**Comment by Kevin-Solomon** [comment]
Thank you for reviewing my code, Ankur

**Pull request merged**